lib/trento/discovery/policies/cluster_policy.ex

```diff
@@ -359,6 +365,21 @@ defmodule Trento.Discovery.Policies.ClusterPolicy do
     ]
   end
 
+  defp parse_maintenance_mode(%{
+         configuration: %{crm_config: %{cluster_properties: cluster_properties}}
+       }) do
+    maintenance_mode_enabled?(
+      parse_crm_cluster_property(
+        cluster_properties,
+        "maintenance-mode",
+        @default_maintenance_mode
+      )
+    )
+  end
+
+  defp maintenance_mode_enabled?(@default_maintenance_mode), do: false
```

Comment:
This code might work, but i find it un secure.
It wouldn't be better:
```
defp maintenance_mode_enabled("true"), do: true
defp maintenance_mode_enabled?(_), do: false
```
At the end, the positive part looks more important than the negative one. I don't know how pacemaker code works, but if it was me who did it, i would be checking the true part there rather than the negative part.
Anyway, it would work the same way

Reply:
done!